wordpress-mobile/WordPress-iOS
# Enable native editor onboarding features globally

## Related PRs

* https://github.com/WordPress/gutenberg/pull/35201
* https://github.com/wordpress-mobile/gutenberg-mobile/pull/4047
* https://github.com/wordpress-mobile/WordPress-Android/pull/15425

## Description

We decided to enable the native editor onboarding features for all users, e.g. the new block badge.

To test: 

In addition to the following, see https://github.com/WordPress/gutenberg/pull/35201. 

1. Install the PR build. 
2. Launch the block editor. 
3. Close the block editor. 
4. ℹ️ **Expected:** The `wpios_editor_session_start` and `wpios_editor_session_end` events fire in Tracks with expected properties:
    * Includes `blogid` 
    * Excludes `can_view_editor_onboarding`

## Regression Notes
1. Potential unintended areas of impact
  The block editor or related analytic events may break. 
2. What I did to test those areas of impact (or what existing automated tests I relied on)
  I manually verified the block editor and editor session events functioned as expected. 
3. What automated tests I added (or what prevented me from doing so)
  This PR focused on code removal, not feature additions or bug fixes. 

PR submission checklist:
